src/steps/privateca/steps/fetchAuthorityCertificates.test.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  executeStepWithDependencies,
+  Recording,
+  StepTestConfig,
+} from '@jupiterone/integration-sdk-testing';
+import { invocationConfig } from '../../..';
+import { integrationConfig } from '../../../../test/config';
+import {
+  getMatchRequestsBy,
+  setupGoogleCloudRecording,
+} from '../../../../test/recording';
+import { PrivatecaSteps } from '../constants';
+
+describe(`privateca#${PrivatecaSteps.STEP_PRIVATE_CA_CERTIFICATES.id}`, () => {
+  let recording: Recording;
```

Comment:
tip: this suggestion will prevent you from ever forgetting the condition in front of `if (recording) await recording.stop()`

```suggestion
  let recording: Recording | undefined;
```